### PR TITLE
Update OpenTelemetry.Instrumentation.W3cTraceContext.Tests to use fil…

### PR DESCRIPTION
**Pull request by jarosal**
…escoped namespaces

Towards #4640 

## Changes

Update project OpenTelemetry.Instrumentation.W3cTraceContext.Tests to use file scoped namespaces.

## Merge requirement checklist

* [X] [CONTRIBUTING](https://github.com/open-telemetry/opentelemetry-dotnet/blob/main/CONTRIBUTING.md) guidelines followed (nullable enabled, static analysis, etc.)
* [X] Unit tests added/updated
* [X] Appropriate `CHANGELOG.md` files updated for non-trivial changes
* [X] Changes in public API reviewed (if applicable)

### CODE REVIEW

File: test/OpenTelemetry.Instrumentation.W3cTraceContext.Tests/W3CTraceContextTests.cs [hunk 1]
```diff
@@ -25,6 +25,9 @@
 using Xunit;
 using Xunit.Abstractions;
 
+
+/* Niescalona zmiana z projektu „OpenTelemetry.Instrumentation.W3cTraceContext.Tests(net6.0)”
```
Comment (reyang) [inline test/OpenTelemetry.Instrumentation.W3cTraceContext.Tests/W3CTraceContextTests.cs:29]: What's this?

Comment (jarosal) [reply test/OpenTelemetry.Instrumentation.W3cTraceContext.Tests/W3CTraceContextTests.cs:29]: Oh, some dotnet format shenanigans, let me fix this

**Review by reyang — APPROVED** [review]

File: test/OpenTelemetry.Instrumentation.W3cTraceContext.Tests/W3CTraceContextTests.cs (hunk 1 above)
Comment (jarosal) [reply test/OpenTelemetry.Instrumentation.W3cTraceContext.Tests/W3CTraceContextTests.cs]: Fixed, weird issue and sorry I didn't catch it (too many PRs to look at)

**Comment by utpilla** [comment]
@jarosal Could you fix the CI check?

**Comment by jarosal** [comment]
@utpilla weird issue... I've removed the whitespace which caused this issue

**Review by utpilla — APPROVED** [review]

**Pull request merged**